src/reducers/visible_filters.js

```diff
@@ -1,5 +1,11 @@
-const visibleFilters = (state = [], action) => {
-  return state
+const visibleFilters = (state = [], {type, payload}) => {
```

Comment:
Having played with how this works, I think the default filters should only be set on initial load - as it is you can't get to the state of no visible filters in the UI, which is confusing I think.

I would change initial state here to `null` or `undefined` and then change https://github.com/mojodna/mapfilter/blob/82f558b8430701eb9cdaca9429fee1aae51e0636/src/selectors/visible_filters.js#L20-L24 to:

```
if (!visibleFilters) {
  visibleFilters = []
  if (dateFieldName) visibleFilters.push(dateFieldName)
  if (bestFilterFields[0]) visibleFilters.push(bestFilterFields[0].fieldname)
  return visibleFilters
}
```